releases.json

```diff
@@ -1540,6 +1540,14 @@
       "1.8.4-1"
     ]
   },
+  "kafel": {
+    "dependency_names": [
+      "kafel"
+    ],
+    "versions": [
+      "2023.10.10-1"
```

Comment:
This doesn't match the archive name, which is 20231004

Reply:
Done

Reply:
Looks like the CI is happy with what you have, so that's fine for me.